static/app/views/insights/common/components/widgets/httpDomainSummaryDurationWidget.tsx

```diff
@@ -0,0 +1,33 @@
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {InsightsLineChartWidget} from 'sentry/views/insights/common/components/insightsLineChartWidget';
+import {useHttpDomainSummaryFilter} from 'sentry/views/insights/common/components/widgets/hooks/useHttpDomainSummaryFilter';
+import {useSpanMetricsSeries} from 'sentry/views/insights/common/queries/useDiscoverSeries';
+import {getDurationChartTitle} from 'sentry/views/insights/common/views/spans/types';
+import {Referrer} from 'sentry/views/insights/http/referrers';
+import {SpanMetricsField} from 'sentry/views/insights/types';
+
+export default function HttpDomainSummaryDurationWidget() {
+  const chartFilters = useHttpDomainSummaryFilter();
+  const {
+    isPending: isDurationDataLoading,
+    data: durationData,
+    error: durationError,
```

Comment:
probably don't need these aliases? maybe makes it easier to read the diffs as we're transitioning

Reply:
Gonna leave them for now, we can go back to update it later